### PR TITLE
Add metrics advisor new package to package index

### PR DESCRIPTION
**Pull request by samvaity**
This PR adds metrics advisor to package index for Java, JS and python.

.NET is still waiting to release.

### CODE REVIEW

File: _data/releases/latest/python-packages.csv [hunk 1]
```diff
@@ -7,6 +7,7 @@
 "azure-eventhub-checkpointstoreblob","1.1.1","","Event Hubs - Azure Blob Storage Checkpoint Store","Event Hubs","eventhub","","","client","",""
 "azure-eventhub-checkpointstoreblob-aio","1.1.1","","Event Hubs - Azure Blob Storage Checkpoint Store AIO","Event Hubs","eventhub","","","client","",""
 "azure-ai-formrecognizer","3.0.0","","Form Recognizer","Form Recognizer","formrecognizer","","","client","",""
+"azure-search-documents","","1.0.0b1","Metrics Advisor","Metrics Advisor","metricsadvisor","","","client","",""
```
Comment (weshaggard) [inline _data/releases/latest/python-packages.csv:10]:
```suggestion
"azure-ai-metricsadvisor","","1.0.0b1","Metrics Advisor","Metrics Advisor","metricsadvisor","NA","NA","client","",""
```

File: _data/releases/latest/java-packages.csv [hunk 2]
```diff
@@ -10,6 +10,7 @@
 "azure-messaging-eventhubs","com.azure","5.1.2","5.2.0-beta.2","Event Hubs","Event Hubs","eventhubs","","","client","","Replaces: azure-eventhubs"
 "azure-messaging-eventhubs-checkpointstore-blob","com.azure","1.1.2","1.2.0-beta.2","Event Hubs - Azure Blob Storage Checkpoint Store","Event Hubs","eventhubs","","","client","",""
 "azure-ai-formrecognizer","com.azure","3.0.0","","Form Recognizer","Form Recognizer","formrecognizer","","","client","",""
+"azure-ai-metricsadvisor","com.azure","","1.0.0-beta.1","Metrics Advisor","Metrics Advisor","metricsadvisor","NA","NA","client","",""
```
Comment (weshaggard) [inline _data/releases/latest/java-packages.csv:13]:
```suggestion
"azure-ai-metricsadvisor","com.azure","","1.0.0-beta.1","Metrics Advisor","Metrics Advisor","metricsadvisor","NA","","client","",""
```

Comment (weshaggard) [reply _data/releases/latest/java-packages.csv:31]: I think we should have github.io docs for this package so we should still remove the NA for GHDocs column.

Comment (samvaity) [reply _data/releases/latest/java-packages.csv]: done.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**